src/GuiBase/Viewer/Gizmo/RotateGizmo.cpp

```diff
@@ -140,7 +160,9 @@ Core::Transform RotateGizmo::mouseMove( const Engine::Camera& cam, const Core::V
                                             m_worldTo * rotationAxis, hits2 );
 
         Core::Vector2 nextXY_ = nextXY;
-        if ( hit1 && hit2 )
+        Scalar angle;
+        // standard check  +  guard against precision issues
+        if ( hit1 && hit2 && hits1[0] > 0.2 && hits2[0] > 0.2 )
```

Comment:
Why the constant ``0.2`` ? This is a very big epsilon ....

Reply:
The thing is that when fitting the camera, some rotation planes are orthogonal to the image plane.
However, the camera doesn't lie exactly on the plane, which makes the rayCast hit the plane very close to the camera when moving the mouse below the plane.
The more you zoom towards the object, the more this value will grown. I found out that at the most zoomed, the value is at 0.6

Reply:
Can someone test the rotation gizmo an very small/large scale scenes to check this "add hoc" value  is ok?

Reply:
Some crash in tangent situations, fixed by commit ff4ed82